css/styles.css

```diff
@@ -96,12 +96,12 @@ them from the MagicMirror² core system and other modules.
   /* Two keyframe positions each to generate a little delay between animations. */
   0%,
   30% {
-    transform: translate(0%, 0);
+    transform3d: translate(0%, 0, 0);
```

Comment:
Shouldn't it be `transform: translate3d(0%, 0, 0);` instead of `transform3d: translate(0%, 0, 0);`?

Reply:
Oh, you're right, i'll fix that